catalog/build.gradle.kts

```diff
@@ -31,6 +31,9 @@ plugins {
 android {
     namespace = "com.adevinta.spark.catalog"
     defaultConfig.applicationId = "com.adevinta.spark.catalog"
+    defaultConfig.resourceConfigurations.addAll(
+        setOf("en-rGB", "fr", "de"),
```

Comment:
I'm not sure how en-us is visible on the list while not being present in this list 🤔

Reply:
Also, I don't see any en-gb string resources in the codebase.

Reply:
After searching around the "English (United States)" is suggested because it's "popular" 🤷
If a user select it, he will still use the en-GB since it's the only english local available (in this context)